pkg/cmd/cli/describe/deployments.go

```diff
@@ -93,19 +93,30 @@ func (d *DeploymentConfigDescriber) Describe(namespace, name string, settings kc
 			if err == nil {
 				sorted := deploymentsHistory.Items
 				sort.Sort(sort.Reverse(rcutils.OverlappingControllers(sorted)))
+				counter := 0
 				for _, item := range sorted {
 					if item.Name != deploymentName && deploymentConfig.Name == deployutil.DeploymentConfigNameFor(&item) {
 						header := fmt.Sprintf("Deployment #%d", deployutil.DeploymentVersionFor(&item))
 						printDeploymentRc(&item, d.kubeClient, out, header, false)
+						counter++
+					}
+					// Only show last three deployments here
+					if counter == 2 {
```

Comment:
Add a comment that this number+1 needs to be on par with the respective number in [oc status](https://github.com/openshift/origin/blob/662753c7a7ee9efbbf3dbe1dd4f4bc324b976d74/pkg/cmd/cli/describe/projectstatus.go#L482)